sentences/nb/light_HassLightSet.yaml

```diff
@@ -2,4 +2,22 @@ language: nb
 intents:
   HassLightSet:
     data:
-      - sentences: []
+      - sentences:
+        - "sett <navn> [til] [lysstyrke] [til] <lysstyrke>"
+        - "sett lysstyrke[n] <i_på> <navn> til <lysstyrke>"
+        response: brightness
+      - sentences:
+          - "sett lysstyrke[n] <i_på> <område> til <lysstyrke>"
+          - "sett <område> [til] [lysstyrke[n]] [til] <lysstyrke>"
+        slots:
+          name: all
+        response: brightness
+      - sentences:
+          - "sett <navn> farge til <farge>"
+          - "sett [farge[n] <i_på>] <navn> til <farge>"
+        response: color
+      - sentences:
+          - "sett <farge>[t | tt] lys <i_på> <område>"
```

Comment:
We meed to remove whitespaces around |, stated in new hassil https://github.com/home-assistant/intents/pull/830.